src/mapgen/mapgen_v6.cpp

```diff
@@ -132,6 +132,39 @@ MapgenV6::MapgenV6(MapgenV6Params *params, EmergeManager *emerge)
 		c_stair_cobble = c_cobble;
 	if (c_stair_desert_stone == CONTENT_IGNORE)
 		c_stair_desert_stone = c_desert_stone;
+
+	if (c_stone == CONTENT_IGNORE)
+		errorstream << "Mapgen v6: Mapgen alias 'mapgen_stone' is invalid!" << std::endl;
+	if (c_dirt == CONTENT_IGNORE)
+		errorstream << "Mapgen v6: Mapgen alias 'mapgen_dirt' is invalid!" << std::endl;
+	if (c_dirt_with_grass == CONTENT_IGNORE)
+		errorstream << "Mapgen v6: Mapgen alias 'mapgen_dirt_with_grass' is invalid!" << std::endl;
+	if (c_water_source == CONTENT_IGNORE)
+		errorstream << "Mapgen v6: Mapgen alias 'mapgen_water_source' is invalid!" << std::endl;
+	if (c_lava_source == CONTENT_IGNORE)
+		errorstream << "Mapgen v6: Mapgen alias 'mapgen_lava_source' is invalid!" << std::endl;
+	if (c_gravel == CONTENT_IGNORE)
+		errorstream << "Mapgen v6: Mapgen alias 'mapgen_gravel' is invalid!" << std::endl;
+	if (c_desert_sand == CONTENT_IGNORE)
+		errorstream << "Mapgen v6: Mapgen alias 'mapgen_desert_sand' is invalid!" << std::endl;
+	if (c_desert_stone == CONTENT_IGNORE)
+		errorstream << "Mapgen v6: Mapgen alias 'mapgen_desert_stone' is invalid!" << std::endl;
+	if (c_dirt_with_snow == CONTENT_IGNORE)
+		errorstream << "Mapgen v6: Mapgen alias 'mapgen_dirt_with_snow' is invalid!" << std::endl;
+	if (c_snow == CONTENT_IGNORE)
+		errorstream << "Mapgen v6: Mapgen alias 'mapgen_snow' is invalid!" << std::endl;
+	if (c_snowblock == CONTENT_IGNORE)
+		errorstream << "Mapgen v6: Mapgen alias 'mapgen_snowblock' is invalid!" << std::endl;
+	if (c_ice == CONTENT_IGNORE)
+		errorstream << "Mapgen v6: Mapgen alias 'mapgen_ice' is invalid!" << std::endl;
+	if (c_cobble == CONTENT_IGNORE)
+		errorstream << "Mapgen v6: Mapgen alias 'mapgen_cobble' is invalid!" << std::endl;
+	if (c_mossycobble == CONTENT_IGNORE)
+		errorstream << "Mapgen v6: Mapgen alias 'mapgen_mossycobble' is invalid!" << std::endl;
+	if (c_stair_cobble == CONTENT_IGNORE)
+		errorstream << "Mapgen v6: Mapgen alias 'mapgen_stair_cobble' is invalid!" << std::endl;
+	if (c_stair_desert_stone == CONTENT_IGNORE)
+		errorstream << "Mapgen v6: Mapgen alias 'mapgen_stair_desert_stone' is invalid!" << std::endl;
```

Comment:
Due to fallbacks, only these mapgen nodes need checking:

stone
dirt
dirt with grass
sand
water source
lava source
cobble

Reply:
Done.